cmd/finch/main_native.go

```diff
@@ -79,6 +79,9 @@ var newApp = func(
 	// TODO: Decide when to forward --debug to the dependencies
 	// (e.g. nerdctl for container commands and limactl for VM commands).
 	rootCmd.PersistentFlags().Bool("debug", false, "running under debug mode")
+	rootCmd.PersistentFlags().String("namespace", "", `containerd namespace, such as "moby" for Docker, "k8s.io" for Kubernetes`)
```

Comment:
It's a bit odd to reference these other namespaces instead of documenting the default "finch" namespace.